src/main/java/org/noteskeeper/server/IGreetingService.java

```diff
@@ -0,0 +1,5 @@
+package org.noteskeeper.server;
+
+public interface IGreetingService {
```

Comment:
I would prefer to use plural form for interface(GreetingServices) and singular for implementation instead of using such suffixes and prefixes. Just like an idea.